server.js

```diff
@@ -151,6 +151,17 @@ if (config.IS_DEV) {
   app.use(development.setUserLocations(config.USER_LOCATIONS))
 }
 
+// Session injection middleware
+app.all('*', function (req, res, next) {
+  req.session.featureFlags = config.FEATURE_FLAGS
+  next()
+})
+
+if (config.SESSION_INJECTION_ENABLED) {
```

Comment:
Only if the session injection feature is enabled, do we want to use these routes.

One thing to think about for the future, is do we want to have this functionality on production in order to test in production without requiring a restart for environment variables. However, without a separate single source of truth for this configuration, to enable the feature in production for everyone will require changing environment variables and restarting.